build/nsm/docker/Test.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM golang:1 as build
```

Comment:
I had started this earlier, but never got around to finishing it nor pushing out a PR, happy to see you did this! :)

Reply:
And now we always use the latest stable go :)